### DISCUSSION TITLE
Video: Setting up Development Environment

### DISCUSSION THREAD

**Discussion by polatengin** [microsoft/vsts-extension-retrospectives#109] [Show and tell]
There are three main ways to start contributing to the Azure DevOps Retrospective Extension.

In the below video all three of them are explained. Please take a look before setting up your development environment for Retrospective Extension.

[![Setting up Development Environment for Retrospective Extension](https://img.youtube.com/vi/GwsscfcAjg8/0.jpg)](https://youtu.be/GwsscfcAjg8)

https://youtu.be/GwsscfcAjg8

**Comment by iivel** [comment]
In a windows dev environment after rolling back to Node 12.x so that the --rev-version switch works when creating the extension I still have an error with the wildcard for ./dist/*.vsix rather than the full file path (which I don't know when using --rev-version) where it says the file does not exist. Any ideas or suggestions on what might be wrong? Thanks!
